apps/web/src/components/molecules/CompanyAdListItem/CompanyAdListItem.tsx

```diff
@@ -1,37 +1,77 @@
-import { Badge } from "@dotkomonline/ui"
+import { type JobListing } from "@dotkomonline/types"
+import { Badge, Icon } from "@dotkomonline/ui"
 import { format } from "date-fns"
 import Image from "next/image"
 import { type FC } from "react"
-import { type CareerAd } from "@/api/get-career-ads"
-
 interface CompanyAdListItemProps {
-  career: CareerAd
+  career: JobListing
 }
+function opprettelseTidspunkt(date: Date){  
+  if (Math.floor((Date.now() - date.getTime()) / (1000 * 60 * 60)) < 1) {
+    return <div>nylig lagt til</div> 
```

Comment:
I think it would be nice to rewrite this function to return only a string and not jsx. By doing this, you make it really easy to write some tests for the function. We can make the tests together next time, but you can rewrite the function to return a string now